src/Tribe/Views/V2/Views/Traits/HTML_Cache.php

```diff
@@ -0,0 +1,192 @@
+<?php
+/**
+ * Provides methods for HTML caching for views
+ *
+ * @since   4.9.11
+ *
+ * @package Tribe\Events\Views\V2\Views
+ */
+
+namespace Tribe\Events\Views\V2\Views\Traits;
+
+use Tribe__Cache_Listener as Cache_Listener;
+use Tribe__Context as Context;
+use Tribe__Date_Utils as Dates;
+
+/**
+ * Trait HTML_Cache
+ *
+ * @since   5.0.0
+ *
+ * @package Tribe\Events\Views\V2\Views
+ *
+ * @property Context $context The current View context.
+ */
+trait HTML_Cache {
+	/**
+	 * Return cached HTML if enabled and cache is set.
+	 *
+	 * @since 5.0.0
+	 *
+	 * @return null|string
+	 */
+	public function maybe_get_cached_html() {
+
+		if ( ! $this->should_cache_html() ) {
+			return;
+		}
+
+		$cache_key = $this->get_cache_html_key();
+
+		if ( $cached_html = tribe( 'cache' )->get_transient( $cache_key, $this->cache_html_triggers() ) ) {
+			return $cached_html;
```

Comment:
:no_entry_sign: Assignments must be the first block of code on a line

<sub>hash: 4da06af6c8c2aff52497271ed1b8d6d3</sub>